src/jit/lsra.cpp

```diff
@@ -3007,14 +3225,18 @@ LinearScan::buildRefPositionsForNode(GenTree *tree,
         {
             if (produce != 0)
             {
-                stack->Push(LocationInfo(currentLoc, interval, tree));
+                LocationInfoList list(listNodePool.GetNode(currentLoc, interval, tree));
+                bool added = operandToLocationInfoMap.AddOrUpdate(tree, list);
+                (assert(added), (void)added);
```

Comment:
I gather that the `(void)added` is to avoid a warning of an unused variable. It could use a comment - I'm pretty sure I will forget this and puzzle over it again in the future without such a comment.